### ISSUE TITLE
UI Tab Ordering

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#453]
Relates to #448 

Tab ordering for username/password and request account link -  Current order is username field, request account link, password field, then login button. It should be username, password, then login button with request account link after the username/password fields?

**Comment by elopatin-uc3** [comment]
I'm reading this question on the Merritt Accessibility spreadsheet as more of a suggestion. The suggested ordering makes sense: username, password, followed by login button with request account link
